typescript/cli/src/validator/utils.ts

```diff
@@ -38,21 +42,21 @@ export const getValidatorStorageLocations = async (
 export const getLatestValidatorCheckpointIndexAndUrl = async (
   s3StorageLocation: string,
 ): Promise<[number, string] | undefined> => {
-  let s3Validator: S3Validator;
+  let validator: S3Validator | GcpValidator;
```

Comment:
use BaseValidator instead
```suggestion
  let validator: BaseValidator;
```